Tests/HTMLEntitiesTests/HTMLEntitiesTests.swift

```diff
@@ -33,11 +33,6 @@ let str3Escaped = "Jako efektivn&ecaron;j&scaron;&iacute; se n&aacute;m jev&iacu
 
 class HTMLEntitiesTests: XCTestCase {
     func testNamedCharacterReferences() {
-#if os(Linux)
-        XCTAssertEqual(namedCharactersEncodeMap.count, 1367)
-#else
-        XCTAssertEqual(namedCharactersEncodeMap.count, 1509)
-#endif
```

Comment:
Instead of removing this check, add a version check so Swift 4.2 would check against the 1509 count instead of 1367. Like so:
```swift
#if os(Linux) && !swift(>=4.2)
        XCTAssertEqual(namedCharactersEncodeMap.count, 1367)	
#else	
        XCTAssertEqual(namedCharactersEncodeMap.count, 1509)	
#endif
```